util/design/gen-flash-img.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Takes a compiled vmem image and processes it for flash.
+    Long term this should include both layers of ECC and scrambling,
+    The first version has only the truncated plaintext ECC.
+"""
+
+import re
+from pathlib import Path
+import argparse
+import secded_gen
+import math
```

Comment:
Nit: you could hit this file (and the one below) with isort (i.e., `isort util/design/gen-flash-img.py`) to sort these.

Reply:
oo i didnt know we had that. Let me do that.